docs/source/policies/policies.md

```diff
@@ -8,7 +8,7 @@ In this topic, we'll cover:
 * [What is a policy](#what-is-a-policy)
 * [Why are policies needed](#why-are-policies-needed)
 * [How are policies implemented throughout Fabric](#how-are-policies-implemented-throughout-fabric)
-* [Fabric policy domains](#fabric-policy-domains)
+* [Fabric policy domains](#the-fabric-policy-domains)
 * [How do you write a policy in Fabric](#how-do-you-write-a-policy-in-fabric)
 * [Fabric chaincode lifecycle](#fabric-chaincode-lifecyle)
```

Comment:
Link does not work because of typo in #fabric-chaincode-lifecyle needs the last 'c' in the word 'lifecycle'

Reply:
I can fix or you can.

Reply:
Technically not part of the original change, but found during review.

Reply:
going to fix this in another PR. Closing this one.